frontend/src/index.tsx

```diff
@@ -22,12 +24,18 @@ if (typeof window !== 'undefined') {
     }
 }
 
+const options = {
+    api_host: window.JS_POSTHOG_HOST,
+}
+
 function renderApp(): void {
     const root = document.getElementById('root')
     if (root) {
         ReactDOM.render(
             <ErrorBoundary>
-                <App />
+                <PostHogProvider apiKey={window.JS_POSTHOG_API_KEY} options={options}>
```

Comment:
We should just provide the initialised posthog from `loadPostHogJS()`, rather than create a new one here. So: 

```
import posthog from 'posthog-js';
<PostHogProvider client={posthog}>
```